src/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -559,13 +565,32 @@ private async Task<ActionResult> AuthenticateIdPortenToken(string originalToken)
 
                 string issuer = _generalSettings.AltinnOidcIssuerUrl;
 
+                string authLevelValue = "0";
+
+                if (authLevel.Equals(IdportenLevel0))
```

Comment:
Do we trust authLevel from the token to always be lower case?

Reply:
Yes. We trust the documentation :-)